### PR TITLE
[Lore] Renames the Sif Homeguard to Sif Defense Force

### PR DESCRIPTION
**Pull request by Anewbe**

### CODE REVIEW

File: code/modules/busy_space/organizations.dm [hunk 1]
```diff
@@ -326,12 +326,12 @@
...
 
 /datum/lore/organization/mil/sif_guard
-	name = "Sif Homeguard Forces" // Todo: Get better name from lorepeople.
+	name = "Sif Defense Force" // Todo: Get better name from lorepeople.
 	short_name = "SifGuard"
 	desc = ""
 	history = ""
 	work = "Sif Governmental Authority's military"
-	headquarters = "Sif" // Make this more specific later.
+	headquarters = "New Reykjavik" // Make this more specific later.
```
Comment (Belsima) [inline code/modules/busy_space/organizations.dm]: Do you need the comment on specificity anymore?

Comment (Anewbe) [inline code/modules/busy_space/organizations.dm]: Probably not.

**Comment by OhEmGee** [comment]
~~you're doing this just to spite me, aren't you?~~

**Comment by Anewbe** [comment]
No, because this change is in line with the written lore.

**Comment by Anewbe** [comment]
Also removes the Sol Home Guard beret, it's just a red beret and we'll hopefully have a fancier one to replace it soon.

**Pull request merged**